www/config.yml

```diff
@@ -179,6 +179,15 @@ chapters:
       twitter: pyladiesLA
       meetup: Pyladies-LA
 
+    chapters:
```

Comment:
@dynarro The `chapters:` label is only needed at the very beginning of the list of chapters. Line 182 should be removed. If you need help submitting an update to this pull request, please let us know :)